server/controllers/Conversations.js

```diff
@@ -205,16 +201,15 @@ module.exports.leaveConversation = async (req, res) => {
         // if a successor was chosen
         if(successor_id){
             // set successor's is_admin value to true
-            let successor = Participant.update({
+            // let successor = // not sure a value could have been caught here
```

Comment:
Yeah. This is unnecessary, even if a value is cought it's not used anywhere else.